### PR TITLE
Reintroduce #257

### PR DESCRIPTION
**Pull request by Kojoley**

### CODE REVIEW

**Comment by djowel** [comment]
@denzor200 I'm going to merge this after checks have completed. But please comment if you will.

File: include/boost/fusion/view/transform_view/detail/deref_data_impl.hpp [hunk 1]
```diff
@@ -17,6 +17,11 @@
 #include <boost/type_traits/is_const.hpp>
 #include <boost/utility/result_of.hpp>
 #include <boost/mpl/if.hpp>
+#include <boost/type_traits/add_const.hpp>
```
Comment (denzor200) [inline include/boost/fusion/view/transform_view/detail/deref_data_impl.hpp:20]: redundantly changes in deref_data_impl.hpp

**Review by denzor200 — CHANGES_REQUESTED** [review]

**Comment by Kojoley** [comment]
@djowel could you please merge develop to master to include this in the next release?

**Comment by djowel** [comment]
Merged.

**Pull request merged**